<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Spellbook</class>
 <widget class="QMainWindow" name="Spellbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spellbook</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="spellList">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>431</width>
      <height>671</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>175</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>50</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Chancery L</family>
      <pointsize>30</pointsize>
      <weight>7</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Vivaldi&quot;;
font: 57 italic 30pt &quot;URW Chancery L&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="schoolLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>136</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rangeLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>201</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="concentrationLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>231</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ritualLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>228</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="durationLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>261</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="componentsLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>291</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="materialLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>321</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="levelLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>137</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>471</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="sort1Box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sort1Label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort field 1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sort2Box">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sort2Label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort field 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="classesLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>381</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="subclassesLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>411</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by class</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="descScrollArea">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>471</y>
      <width>591</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="descriptionTitle">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>441</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="castingTimeLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pageLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>140</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="favoritesButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Favorites</string>
    </property>
   </widget>
   <widget class="QPushButton" name="favButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>50</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
